### PR TITLE
fix: texts and links

### PR DESCRIPTION
**Pull request by zxyanliu**
## Description

* Refined UI texts (help contents). 
* Updated some aka links (help links). 
* Ran `yarn l10n` script and committed resulting changes to en-US.json.
* Followed suggested [guidelines](https://github.com/microsoft/BotFramework-Composer/blob/main/docs/guidelines/Composer-help-contents.md) to make the changes

## Task Item

https://github.com/microsoft/BotFramework-Composer/issues/4508
#minor

#### Before 

image

#### After 
image

#### Before 

image

image

#### After 

image
Learn more points to: https://aka.ms/lu-file-format

image
Learn more points to: https://aka.ms/lg-file-format

#### Before 

image

#### After 

image
Added learn more points to: https://aka.ms/composer-multilingual-support

#### Before 

image

#### After

image

#### Before

image

image

image

image

#### After

image

image

image

image

### CODE REVIEW

**Comment by zxyanliu** [comment]
cc: @mareekuh @garypretty

File: Composer/packages/client/src/components/CreationFlow/CreateOptions.tsx [hunk 1]
```diff
@@ -261,7 +261,7 @@ export function CreateOptions(props: CreateOptionsProps) {
         onDismiss={onDismiss}
       >
         <ChoiceGroup
-          label={formatMessage('Choose how to create your bot')}
+          // label={formatMessage('Choose how to create your bot')}
```
Comment (boydc2014) [inline Composer/packages/client/src/components/CreationFlow/CreateOptions.tsx:264]: prefer just remove it if we don't need it. git will remember it if we need it back later

**Comment by boydc2014** [comment]
Maybe too much to ask, but do you have a picture of before and after?

File: Composer/packages/client/src/components/CreationFlow/CreateOptions.tsx (hunk 1 above)
Comment (zxyanliu) [reply Composer/packages/client/src/components/CreationFlow/CreateOptions.tsx:264]: No problem!

**Comment by zxyanliu** [comment]
> Maybe too much to ask, but do you have a picture of before and after?

I can add them.

**Comment by zxyanliu** [comment]
Closing this issue for now. Will reopen when necessary.

**Pull request closed without merging**